axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -1817,6 +1817,7 @@
     "gasPrice": "0.00005uamplifier",
     "gasLimit": "auto",
     "govProposalDepositAmount": "100000000",
+    "govProposalExpeditedDepositAmount": "150000000",
```

Comment:
you mentioned that expedited proposals can't be used right away. so custom devnets need to have their corresponding params updated?